prebuilts-buildbot.py

```diff
@@ -78,7 +78,7 @@ def get_buildtype():
     return os.environ.get('BUILDTYPE', 'Debug')
 
 def get_build_edition():
-    return os.environ.get('BUILD_EDITION', 'community')
+    return 'community' #os.environ.get('BUILD_EDITION', 'community')
```

Comment:
This will need to be reverted on `develop` and or sorted out properly in the unlikely event we introduce any commercial prebuilts we need to use `develop-9.0`